### DISCUSSION TITLE
Grouping Speakers - error when trying to group with Sonos Roam

### DISCUSSION THREAD

**Discussion by nbanbury** [punxaphil/custom-sonos-card#267] [Q&A] [answered]
Hi there.

When I try to group my Sonos Roam with any other Sonos speaker I get the following error message:
"Failed to call service media_player/join. Error calling SonosSpeaker.join on Gallery: UPnP Error 501 received: Action Failed from <IP ADDRESS>"

I also get the same issue if I am playing music on my Roam and try to group from there.
"Failed to call service media_player/join. Error calling SonosSpeaker.join on Sonos Roam: UPnP Error 501 received: Action Failed from <IP Address>"

Grouping across my other Sonos speakers works fine.

Is there something I need to update to fix this?

Thanks

**Comment by punxaphil** [comment]
It works in the official Sonos app?

**Reply by nbanbury** [reply]
Yes - and when I logged into that app I was told Sonos Roam needed an update. And now it works fine. Sorry! should have checked that first!

**Comment by nbanbury** [comment, marked as answer]
Resolved with update to Sonos Roam
